tests/wamr-test-suites/test_wamr.sh

```diff
@@ -332,6 +332,9 @@ function spec_test()
     if [[ ${ENABLE_SIMD} == 1 ]]; then
         git apply ../../spec-test-script/simd_ignore_cases.patch
     fi
+    if [[ ${ENABLE_MULTI_MODULE} == 1 && $1 == 'aot'  ]]; then
```

Comment:
Not sure whether llvm-jit also supports multi-module? If yes, here we can modify it to somewhat like:
```bash
if [[ ${ENABLE_MULTI_MODULE} == 1 && ($1 == 'aot' || $1 == 'jit') ]]; then
```

Reply:
I think we don't support jit